frontend/src/templates/SignInTemplate.js

```diff
@@ -1,28 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Formik, Field, ErrorMessage } from 'formik';
+import * as yup from 'yup';
+import { Form, Button, Row, Badge } from 'react-bootstrap';
 
-import { SignInForm, Navigation } from 'src/organisms/';
-import { Row, Col, Container } from 'react-bootstrap';
+import { FormikGroup } from '../molecules';
+import { Link } from '../atoms';
```

Comment:
tady to pri kompilaci pada, v atoms Link neni ... nema tu byt jenom `import { Link } from 'react-router-dom';?
`

Reply:
v atoms Link byl, ale už není... předělám :)